tracing-subscriber/src/fmt/mod.rs

```diff
@@ -664,6 +786,43 @@ impl Default for Settings {
     }
 }
 
+/// Install a global tracing subscriber that listens for events and
+/// filters based on the value of the [`RUST_LOG` environment variable],
+/// if one is not already set.
+///
+/// If the `tracing_log` feature is enabled, this will also install
+/// the LogTracer to convert `Log` records into `tracing` `Event`s.
+///
+///
+/// # Errors
+/// Returns an Error if the initialization was unsuccessful,
+/// likely because a global subscriber was already installed by another
+/// call to `try_init`.
+///
+/// [`RUST_LOG` environment variable]:
+///     ../filter/struct.EnvFilter.html#associatedconstant.DEFAULT_ENV
+pub fn try_init() -> Result<(), impl Error + Send + Sync + 'static> {
+    Subscriber::builder()
+        .with_env_filter(EnvFilter::from_default_env())
+        .try_init()
+}
+
+/// Install a global tracing subscriber that listens for events and
+/// filters based on the value of the [`RUST_LOG` environment variable].
+///
+/// If the `tracing_log` feature is enabled, this will also install
+/// the LogTracer to convert `Log` records into `tracing` `Event`s.
```

Comment:
Nit:
```suggestion
/// the `LogTracer` to convert `log` records into `tracing` `Event`s.
```
also, maybe `LogTracer` should be linkified?